app/models/order.rb

```diff
@@ -14,8 +14,6 @@ class Order < ApplicationRecord
 
   enum status: { pending: 0, successful: 1, cancelled: 2 }
 
-  scope :status, ->(status_param) { where(status: status_param) }
-
   private
 
   def pickup_time_cannot_be_in_the_past
```

Comment:
https://github.com/atifjavaid267/Online-Marketplace-for-Selling-Scrape/pull/80/files#diff-26bbec4be8ce33f60fa824e2bd29e6902e57541818578634bea06bce180b620eR32

Should be moved to scope